lb_matching_tools/cleaner.py

```diff
@@ -36,34 +43,108 @@ class MetadataCleaner:
         r"(?P<title>.+?)(?:\s+?(&|with))(?P<dash>.*)",
     ]
 
-    def __init__(self, preferred_script = "Latin"):
+    def __init__(self, preferred_script="Latin"):
         self.recording_expressions = [re.compile(exp, re.IGNORECASE) for exp in self.RECORDING_EXPRESSIONS]
         self.artist_expressions = [re.compile(exp, re.IGNORECASE) for exp in self.ARTIST_EXPRESSIONS]
         self.foreign_script_expression = regex.compile(r"[^\p{Script=Common}\p{Script=" + preferred_script + r"}]+")
+        self.paren_guff_expression = regex.compile(r"(20[0-9]{2}|19[0-9]{2})")
```

Comment:
```suggestion
        self.paren_guff_expression = re.compile(r"(20[0-9]{2}|19[0-9]{2})")
```
No need to use the slower `regex` library for this specific expression here.